hawkular-metrics/hawkular-metrics-wrapper.sh

```diff
@@ -144,6 +120,7 @@ exec 2>&1 /opt/jboss/wildfly/bin/standalone.sh \
   -Djavax.net.ssl.keyStore=$HAWKULAR_METRICS_AUTH_DIR/hawkular-metrics.keystore \
   -Djavax.net.ssl.trustStore=$HAWKULAR_METRICS_AUTH_DIR/hawkular-metrics.truststore \
   -Djboss.node.name=$HOSTNAME \
+  -Djgroups.password=${JGROUPS_PASSWORD} \
```

Comment:
please don't set  the password like this, put it in a property file like we do with the keystore/truststore passwords. This will prevent the password from leaking into things like ps